app/models/manageiq/providers/azure/inventory/collector/cloud_manager.rb

```diff
@@ -35,31 +35,7 @@ def availability_zones
   end
 
   def stacks
-    stacks = collect_inventory(:deployments) { gather_data_for_this_region(@tds, 'list') }
-
-    stacks.each do |deployment|
-      stack_template_hash(deployment)
-    end
-
-    stacks
-  end
-
-  def stack_templates
-    # download all template uris
-    _log.info("Retrieving templates...")
-    @template_uris.each { |uri, template| template[:content] = download_template(uri) }
-    _log.info("Retrieving templates...Complete - Count [#{@template_uris.count}]")
-    _log.debug("Memory usage: #{'%.02f' % collector_memory_usage} MiB")
-
-    # load from existing stacks => templates
-    stacks = OrchestrationStack.where(:ems_ref => @template_refs.keys, :ext_management_system => @ems).index_by(&:ems_ref)
-    @template_refs.each do |stack_ref, template|
-      template[:content] = stacks[stack_ref].try(:orchestration_template).try(:content)
-    end
-
-    (@template_uris.values + @template_refs.values + @template_directs.values).select do |raw|
-      raw[:content]
-    end
+    @stacks_cache ||= collect_inventory(:deployments) { gather_data_for_this_region(@tds, 'list') }
```

Comment:
@Ladas Where is @stacks_cache used?

Reply:
@bronaghs I use instance variable only within a method, so it's just a cache, since this method can be called multiple times